src/game_interpreter_map.h

```diff
@@ -38,6 +38,8 @@ class Game_Interpreter_Map : public Game_Interpreter
 public:
 	Game_Interpreter_Map(int _depth = 0, bool _main_flag = false);
 
+	~Game_Interpreter_Map() {}
```

Comment:
any reason for this?
I bet some obscure corner case specified in the standard almost nobody knows for "C++ will leak this because it is inherited"?